README.md

```diff
@@ -257,7 +257,12 @@ problems are fixable, but some errors must be fixed manually.
 
 ## Can I override the `fix: true` config setting?
 
-Also yes! You can use `standardrb --no-fix`. Not `fix`ing is the default behavior, but this flag will override the `fix: true` setting in your [`.standard.yml` config](#what-you-might-do-if-youre-clever). This is especially useful for checking your projects compliance with `standardrb` in CI environments while keeping the `fix: true` option enabled locally.
+Also yes! You can use `standardrb --no-fix`. Not `fix`ing is the default
```

Comment:
This section changes `projects` to `project's` but also corrects the line formatting to be consistent with the rest of the readme. The typo change isn't clear from the GitHub diff because of the line changes.